### PR TITLE
Feature/full readme

### PR DESCRIPTION
**Pull request by jeevanjoseph**
Updating the README.md, adding CONTRIBUTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,18 +60,29 @@ lb_public_url = http://xxx.xxx.xxx.xxx
...
-## Deployment Topology
+MuShop Complete is a polyglot micro-services application built to showcase a cloud native approach to application development on [Oracle Cloud Infrastructure (OCI)][oci] using Oracle's [cloud native](https://www.oracle.com/cloud/cloud-native/) services. MuShop Complete uses a Kubernetes cluster, and can be deployed using the provided `helm` charts (preferred), or Kubernetes manifests. It is recommended to use an Oracle Container Engine for Kubernetes cluster, however other Kubernetes distributions will also work.
+
+The [helm chart documentation][chartdocs] walks through the deployment process and various options for customizing the deployment  
+
+###  MuShop Complete Topology
 
 The following diagram shows the topology created by this stack.
 
-![MuShop Basic Infra](./images/basic/00-Topology.png)
```
Comment (junior) [inline README.md]: MuShop Basic Infra pointing to complete topology folder

File: README.md [hunk 2]
```diff
@@ -44,18 +60,29 @@ lb_public_url = http://xxx.xxx.xxx.xxx
...
 
-## Questions
+### MuShop Basic Topology
 
-If you have an issue or a question, please take a look at our [FAQs](./deploy/basic/FAQs.md) or [open an issue](https://github.com/oracle/oci-quickstart-cloudnative/issues/new).
+The following diagram shows the topology created by this stack.
+
+![MuShop Basic Infra](./images/basic/00-Free-Tier.png)
+
+## Getting Started with MuShop Complete
```
Comment (junior) [inline README.md]: MuShop Basic Topology and then Getting Stated with MuShop Complete?

Comment (junior) [reply README.md]: We should have a separator between the two

**Comment by junior** [comment]
image

The text "Options for different MuShop deployment modes" is showing as a comment.

File: README.md (hunk 2 above)
Comment (jeevanjoseph) [reply README.md]: done.

File: README.md (hunk 1 above)
Comment (jeevanjoseph) [reply README.md]: its pointing to the right image and folder now.

**Comment by jeevanjoseph** [comment]
@mvandervliet - did you get a chance to look at this ?

**Comment by jeevanjoseph** [comment]
Let me know if there is a way to shorten the intro.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,31 @@
 ![MuShop Logo](./images/logo.png)
 
 ---
-
-This is a Terraform configuration that deploys the MuShop basic sample application on [Oracle Cloud Infrastructure (OCI)][oci]. 
-
-MuShop basic is a 3-tier web application that implements an e-commerce site. It is built to showcase the features of [Oracle Cloud Infrastructure (OCI)][oci]. This application is designed to run using only the Always Free tier resources. 
-The repository contains the application code as well as the [Terraform][tf] code to create a [Resource Manager][orm] stack, 
-that creates all the required resources and configures the application on the created resources.
+MuShop is a showcase of several [Oracle Cloud Infrastructure (OCI)][oci] services in a unified reference application. The sample application implements an e-commerce platform built as a set of micro-services. The accompanying content can be used as a self paced workshop to get started with cloud native application development on [Oracle Cloud Infrastructure (OCI)][oci].
```
Comment (mvandervliet) [inline README.md]: I think we should remove ~as a self paced workshop~ here, as that is not clearly presented elsewhere.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,31 @@
...
 
 | ![home](./images/screenshot/mushop.home.png) | ![browse](./images/screenshot/mushop.browse.png) | ![cart](./images/screenshot/mushop.cart.png) | ![about](./images/screenshot/mushop.about.png) |
 |---|---|---|---|
 
-## Getting Started
+MuShop can be deployed in different ways to explore [Oracle Cloud Infrastructure (OCI)][oci] based on your subscription. Both deployment models can be used with trial subscriptions. However, [Oracle Cloud Infrastructure (OCI)][oci] offers an *Always Free* tier with resources that can be used indefinitely.
```
Comment (mvandervliet) [inline README.md]: we need to leave out `OCI` in all cases

File: README.md [hunk 5]
```diff
@@ -44,18 +58,31 @@ lb_public_url = http://xxx.xxx.xxx.xxx
...
 
 The following diagram shows the topology created by this stack.
 
-![MuShop Basic Infra](./images/basic/00-Topology.png)
+![MuShop Basic Infra](./images/complete/00-Topology.png)
 
-To learn more about how you can use this application to build your own applications, click [here](./deploy/basic/README.md).
+## Questions
+
+If you have an issue or a question, please take a look at our [FAQs](./deploy/basic/FAQs.md) or [open an issue](https://github.com/oracle/oci-quickstart-cloudnative/issues/new).
```
Comment (mvandervliet) [inline README.md]: This URL is the old one, probably should be updated.

File: README.md [hunk 6]
```diff
@@ -1,17 +1,31 @@
...
+└── deploy
+    ├── basic
+    └── complete
+```
+
+## Getting Started with MuShop Basic
+
+This is a Terraform configuration that deploys the MuShop basic sample application on [Oracle Cloud Infrastructure (OCI)][oci].
+
+MuShop Basic is a 3-tier web application that implements an e-commerce site. It is built to showcase the features of [Oracle Cloud Infrastructure (OCI)][oci]. This application is designed to run using only the Always Free tier resources. The services run directly on OCI compute resources and leverage [Oracle Cloud Infrastructure (OCI)][oci] services.
```
Comment (mvandervliet) [inline README.md]: This paragraph seems redundant. Perhaps simply add the "Always Free" qualifier in the first sentence.

File: README.md [hunk 7]
```diff
@@ -1,17 +1,31 @@
...
+
+The repository contains the application code as well as the [Terraform][tf] code to create a [Resource Manager][orm] stack, that creates all the required resources and configures the application on the created resources.
```
Comment (mvandervliet) [inline README.md]: Add here something like: 
> To simplify getting started, the Resource Manager Stack is created as part of each [release](https://github.com/oracle-quickstart/oci-cloudnative/releases)

File: README.md (hunk 4 above)
Comment (junior) [reply README.md]: ??? This is oci-quickstart and both options need oci.

Comment (mvandervliet) [reply README.md]: It's just that the use of "OCI" may be subject to trademark, etc.

**Comment by mvandervliet** [comment]
Bump @jeevanjoseph

**Pull request merged**